mh/arena_utils.py

```diff
@@ -0,0 +1,200 @@
+"""
+Arena Quest ID List:
+EA60 
+EA61
+EA62
+EA63
+EA64
+EA65
+EA66
+EA67
+EA68
+EA69
+EA6A
+EA6B
+"""
+
+# bird and brute bowgun: rathling gun + barrel, poison stinger frame, light bowgun stock
+# sea power bowgun: rathling gun barrel, rathling gun + frame, rathling gun + stock
+# land lords bowgun: jho barrel, diablos frame, barioth stock
+# two flames bowgun: lagiacrus barrel, lagiacrus frame, barioth stock
```

Comment:
These comments are unnecessary here. Feel free to move them closer to the variables holding the quests if you plan to still keep them.